lib/openc_bot/helpers/reporting.rb

```diff
@@ -57,22 +55,39 @@ def report_run_to_analysis_app(params)
         run_params.merge!(bot_id: bot_id, bot_type: "external", git_commit: current_git_commit)
         run_params[:output] ||= params.to_s unless params.blank?
         _analysis_http_post("#{ANALYSIS_HOST}/runs", run: run_params)
-      rescue Exception => e
+      rescue StandardError => e
         puts "Exception (#{e.inspect}) reporting run to analysis app"
       end
 
       # DEPRECATED. Please use report_run_to_analysis_app instead of report_run_to_oc
       alias report_run_to_oc report_run_to_analysis_app
 
-      def report_progress_to_analysis_app(companies_processed:, companies_added: nil, companies_updated: nil)
+      def track_company_processed
+        StatsD.increment("#{statsd_namespace}.processed", sample_rate: 1.0)
+
+        incremeent_progress_counters(companies_processed_delta: 1)
+
+        @last_reported_progress ||= Time.new(0)
+        return unless Time.now > @last_reported_progress + PROGRESS_REPORT_FREQUENCY
+
+        report_progress_to_analysis_app
+        @last_reported_progress = Time.now
+      end
+
+      def incremeent_progress_counters(companies_processed_delta: nil)
```

Comment:
looks like a typo? increment

Reply:
"incremeent" . I'm not sure how I stared at this code for so long and didn't see that.
Fixed ✅